2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Configure stats pid to 28244
2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from C:\Users\inigo\.config\wandb\settings
2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from F:\CV_MATERIAL\M6\WEEK3\detr\wandb\settings
2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': '..\\traindetr.py', 'program': 'f:\\cv_material\\m6\\week3\\traindetr.py'}
2023-03-25 21:52:06,137 INFO    MainThread:28244 [wandb_init.py:_log_setup():506] Logging user logs to F:\CV_MATERIAL\M6\WEEK3\detr\wandb\run-20230325_215206-ssrlgq6u\logs\debug.log
2023-03-25 21:52:06,138 INFO    MainThread:28244 [wandb_init.py:_log_setup():507] Logging internal logs to F:\CV_MATERIAL\M6\WEEK3\detr\wandb\run-20230325_215206-ssrlgq6u\logs\debug-internal.log
2023-03-25 21:52:06,138 INFO    MainThread:28244 [wandb_init.py:_jupyter_setup():452] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000227A0D41F40>
2023-03-25 21:52:06,138 INFO    MainThread:28244 [wandb_init.py:init():546] calling init triggers
2023-03-25 21:52:06,138 INFO    MainThread:28244 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2023-03-25 21:52:06,138 INFO    MainThread:28244 [wandb_init.py:init():602] starting backend
2023-03-25 21:52:06,138 INFO    MainThread:28244 [wandb_init.py:init():606] setting up manager
2023-03-25 21:52:06,142 INFO    MainThread:28244 [wandb_init.py:init():613] backend started and connected
2023-03-25 21:52:06,144 INFO    MainThread:28244 [wandb_run.py:_label_probe_notebook():1202] probe notebook
2023-03-25 21:52:06,144 INFO    MainThread:28244 [wandb_run.py:_label_probe_notebook():1212] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-25 21:52:06,144 INFO    MainThread:28244 [wandb_init.py:init():701] updated telemetry
2023-03-25 21:52:06,175 INFO    MainThread:28244 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-03-25 21:52:06,593 INFO    MainThread:28244 [wandb_run.py:_on_init():2131] communicating current version
2023-03-25 21:52:06,830 INFO    MainThread:28244 [wandb_run.py:_on_init():2140] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-25 21:52:06,830 INFO    MainThread:28244 [wandb_init.py:init():789] starting run threads in backend
2023-03-25 21:52:06,961 INFO    MainThread:28244 [wandb_run.py:_console_start():2112] atexit reg
2023-03-25 21:52:06,961 INFO    MainThread:28244 [wandb_run.py:_redirect():1967] redirect: SettingsConsole.OFF
2023-03-25 21:52:06,961 INFO    MainThread:28244 [wandb_init.py:init():831] run started, returning control to user process
2023-03-25 22:29:34,541 INFO    MainThread:28244 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-25 22:29:34,543 INFO    MainThread:28244 [wandb_init.py:_pause_backend():417] pausing backend
2023-03-25 22:41:36,881 INFO    MainThread:28244 [wandb_init.py:_resume_backend():422] resuming backend
2023-03-25 22:41:44,042 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Configure stats pid to 28244
2023-03-25 22:41:44,043 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from C:\Users\inigo\.config\wandb\settings
2023-03-25 22:41:44,043 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from F:\CV_MATERIAL\M6\WEEK3\detr\wandb\settings
2023-03-25 22:41:44,044 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-25 22:41:44,044 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-03-25 22:41:44,045 INFO    MainThread:28244 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': '..\\traindetr.py', 'program': 'f:\\cv_material\\m6\\week3\\traindetr.py'}
2023-03-25 22:41:44,046 INFO    MainThread:28244 [wandb_init.py:_log_setup():506] Logging user logs to F:\CV_MATERIAL\M6\WEEK3\detr\wandb\run-20230325_224144-58usy3dw\logs\debug.log
2023-03-25 22:41:44,047 INFO    MainThread:28244 [wandb_init.py:_log_setup():507] Logging internal logs to F:\CV_MATERIAL\M6\WEEK3\detr\wandb\run-20230325_224144-58usy3dw\logs\debug-internal.log
2023-03-25 22:41:44,047 INFO    MainThread:28244 [wandb_init.py:init():546] calling init triggers
2023-03-25 22:41:44,048 INFO    MainThread:28244 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2023-03-25 22:41:44,048 INFO    MainThread:28244 [wandb_init.py:init():568] re-initializing run, found existing run on stack: ssrlgq6u
2023-03-25 22:41:44,050 INFO    MainThread:28244 [wandb_run.py:_finish():1852] finishing run mcv-m5-team4/detrFineTune/ssrlgq6u
2023-03-25 22:41:44,333 INFO    MainThread:28244 [jupyter.py:save_history():445] not saving jupyter history
2023-03-25 22:41:44,333 INFO    MainThread:28244 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-03-25 22:41:44,334 INFO    MainThread:28244 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2023-03-25 22:41:44,334 INFO    MainThread:28244 [wandb_run.py:_atexit_cleanup():2081] got exitcode: 0
2023-03-25 22:41:44,336 INFO    MainThread:28244 [wandb_run.py:_restore():2064] restore
2023-03-25 22:41:44,336 INFO    MainThread:28244 [wandb_run.py:_restore():2070] restore done
2023-03-25 22:41:52,638 INFO    MainThread:28244 [wandb_run.py:_footer_history_summary_info():3429] rendering history
2023-03-25 22:41:52,638 INFO    MainThread:28244 [wandb_run.py:_footer_history_summary_info():3461] rendering summary
2023-03-25 22:41:52,655 INFO    MainThread:28244 [wandb_run.py:_footer_sync_info():3387] logging synced files
